java/performance/src/test/java/org/apache/arrow/vector/Float8Benchmarks.java

```diff
@@ -64,14 +73,28 @@ public void prepare() {
     fromVector = new Float8Vector("vector", allocator);
     fromVector.allocateNew(VECTOR_LENGTH);
 
-    for (int i = 0;i < VECTOR_LENGTH; i++) {
+    compareVector1 = new Float8Vector("vector cmp1", allocator);
+    compareVector1.allocateNew(VECTOR_LENGTH);
+    compareVector2 = new Float8Vector("vector cmp2", allocator);
+    compareVector2.allocateNew(VECTOR_LENGTH);
```

Comment:
please add float4 vectors as part of this benchmark.

Reply:
Sure. I will add benchmarks for float4 to class Float4Benchmarks.

Reply:
Please consolidate into 1 floatbenchmarks class and a single benchmark. (reasoning behind this: http://insightfullogic.com/2014/May/12/fast-and-megamorphic-what-influences-method-invoca/)

Reply:
For things not on the hot path it probably isn't worth changing.  The only reason I'm bringing it up here is because the code seemed to already written to avoid those calls.   If there are a lot of places you intend to fix then having a discussion on the mailing list would be a good idea.

Reply:
Sure. Sounds reasonable.